libcxx/test/support/count_new.h

```diff
@@ -460,6 +460,8 @@ inline void* alocate_aligned_impl(std::size_t size, std::align_val_t align) {
   void* ret                   = nullptr;
 #    ifdef USE_ALIGNED_ALLOC
   ret = _aligned_malloc(size, alignment);
+#    elif TEST_STD_VER >= 17
+  ret = aligned_alloc(alignment, size);
```

Comment:
I think we might need to round the size like we do here: https://github.com/llvm/llvm-project/blob/main/libcxx/include/__memory/aligned_alloc.h#L34-L43

Reply:
Done